pkg/controller/seed/reconciler_test.go

```diff
@@ -417,9 +452,8 @@ status: {}
 `
 }
 
-func expectedPDB(namespace string) string {
-
-	return `apiVersion: policy/v1
+func expectedPDB(namespace string, k8sGreaterEqual126 bool) string {
```

Comment:
```suggestion
func expectedPDB(namespace string, k8sVersion *semver.Version ) string {
```

In case a new special case needs to be handled for e.g. k8s 1.28, it does not look convenient to use another boolean parameter, thus pass the k8s version and parse it in the function.

Reply:
I chose to do it this way because if we use the version check here as well, then it just does the same thing as the original function which is tested. But I can change it if you insist.

Reply:
Ok, soudns good.